misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/BlockFormattingOptions.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://wso2.com) All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2023, WSO2 LLC. (http://wso2.com)
```
Make the change for other occurrences as well.